docs/pages/documentation/8-deploy-grakn/grakn-deployment-guide.md

```diff
@@ -47,34 +41,9 @@ We recommend changing this to another location to make upgrading Grakn easier an
 * `commitlog_directory`: eg. /var/lib/cassandra/commitlog
 * `saved_caches_directory`: eg. /var/lib/cassandra/saved_caches
 
-### Clustered Database
-
-To add reliability and redudancy on the database layer, you can run Cassandra (DSE, OrientDB, other storage backends) separately and point Grakn to use the node/cluster.
-
-Running Cassandra requires an entire guide in itself - our brief recommendation is to have, at the minimum, a cluster of 3 nodes with replication factor (RF) 3.
-
-Cassandra version: 2.1.9
-
-Specify an external database cluster in `conf/main/grakn.properties`: `storage.hostname`: eg. 10.0.10.1,10.0.10.2,10.0.10.3
-
-For further information on running Cassandra, see:
-
-- [Cassandra documentation](http://cassandra.apache.org/doc/latest/operating/index.html)    
-- [AWS Cassandra whitepaper](https://d0.awsstatic.com/whitepapers/Cassandra_on_AWS.pdf)
-
 ### Distributed
 
-For a fully scalable and distributed setup, you will need to run each component of Grakn in a separate cluster.
-This provides the highest level of redundancy and availability; you will want this for high throughput systems and to make the most out of Grakn. 
-
-The database layer will need to be configured similarly to the [Clustered database](#clustered-database) setup.
-
-Grakn Engine uses [Redis](https://redis.io/) for distributed task execution.
-
-Redis version: 3.2.9
-
-Once the configuration files are correct, you can start Grakn Engine only with `bin/grakn-engine.sh start`.
-Multiple instances of Grakn Engine can be started that will make use of the shared Cassandra/Redis clusters.
+//TODO this section needs rewording entirely
```

Comment:
@burukuru @lolski We need a way to describe how to make Grakn distributed without mentioning things explicitly. Ideas?

Reply:
@fppt @burukuru Hmm that is tricky, as at the moment we do have to configure Cassandra cluster. Redis at the moment cannot yet be clustered (CMIIW @pluraliseseverythings ).

That's the current state of affair. However let's try to bounce some ideas nonetheless

Reply:
The idea is to start Grakn on each node which will start all the sub components (dbms/storage/queue).
I'm going to work on clustering starting tomorrow so will have a better idea on how to fill out these docs next week.

Reply:
@burukuru yes. we're still lacking that functionality as of now, however